boards.txt

```diff
@@ -57,7 +57,7 @@ feather52832.menu.debug.l1.build.debug_flags=-DCFG_DEBUG=1
 feather52832.menu.debug.l2=Level 2 (Full Debug)
 feather52832.menu.debug.l2.build.debug_flags=-DCFG_DEBUG=2
 feather52832.menu.debug.l3=Level 3 (Segger SystemView)
-feather52832.menu.debug.l3.build.debug_flags=-DCFG_DEBUG=3
+feather52832.menu.debug.l3.build.debug_flags=-DCFG_DEBUG=3 -DCFG_SYSVIEW=0xFFFFFFFF
```

Comment:
why CFG_SYSVIEW has to be max int, an simple 0,1 should be easier to follow.

Reply:
Because longer term, I want to have SysView be a bitmask, so that you could only enable SysView events of specific types.  Already there are lost events with standard JLINK at debug level 3 for any but the simplest projects, and a JLINK EDU Mini is even slower.  So, reducing that output is a longer-term goal, so that it can be more useful by being targeted.  By doing this as 0xFFFFFFFF now, the default at debug level 3 need not be changed, allowing boards.txt to not require a change later.

I hope that explains my thoughts OK for this?

> Please revert the original RTT sources files, we will only use it as it is.

Why revert the RTT sources?  Without those, RTT does not work at all?  What changes of any value would remain?

Reply:
> Because longer term, I want to have SysView be a bitmask, so that you could only enable SysView events of specific types. Already there are lost events with standard JLINK at debug level 3 for any but the simplest projects, and a JLINK EDU Mini is even slower. So, reducing that output is a longer-term goal, so that it can be more useful by being targeted. By doing this as 0xFFFFFFFF now, the default at debug level 3 need not be changed, allowing boards.txt to not require a change later.
> 
> I hope that explains my thoughts OK for this?

I see, that is too advanced, please move the `CFG_SYSVIEW` in to source file, where advanced user can decide which mask they can enable. By default, CFG_DEBUG=3 will also set it to 0xFFFFFFF as current.

> 
> > Please revert the original RTT sources files, we will only use it as it is.
> 
> What do you mean by "revert the original RTT source files"? Without those, RTT does not work at all, as not all RTT files were included before.

I mean we should only use Segger RTT sources file as it is when downloaded from segger. We should not modify it at all, that will make it difficult to upgrade later on.